### PR TITLE
Split mail user/pass for using with ssmtp

### PR DESCRIPTION
**Pull request by diversemix**
Reference https://github.com/libero/reviewer/issues/547

### CODE REVIEW

**Comment by diversemix** [comment]
Re-running tests - not sure why they failed /cc @giorgiosironi  :man_shrugging:

File: salt/elife-xpub/init.sls [hunk 1]
```diff
@@ -53,6 +53,8 @@ elife-xpub-environment-variables-for-configuration:
             export MAILER_PORT={{ pillar.elife_xpub.mailer.port }}
             {% if pillar.elife_xpub.mailer.user %}
             export MAILER_AUTH='{"user": "{{ pillar.elife_xpub.mailer.user }}", "pass":"{{ pillar.elife_xpub.mailer.pass }}"}'
+            export MAILER_AUTH_USER={{ pillar.elife_xpub.mailer.user }}
+            export MAILER_AUTH_PASS={{ pillar.elife_xpub.mailer.pass }}
```
Comment (giorgiosironi) [inline salt/elife-xpub/init.sls]: this is a base64 value so can contain `+`, `/` and `=`. These all seem safe without quoting.

**Comment by giorgiosironi** [comment]
The branch started from an old `master` so it didn't have the fixes in https://github.com/elifesciences/elife-xpub-formula/pull/78, retrying.

File: salt/elife-xpub/init.sls (hunk 1 above)
Comment (diversemix) [reply salt/elife-xpub/init.sls]: Ah - do you think I should quote just to be on the safe side?

**Pull request merged**